utils/src/experimenter.ts

```diff
@@ -14,15 +13,40 @@ export interface ExperimenterProfile {
 
 /** Experimenter data (written to Firestore under experimenterData/{id}). */
 export interface ExperimenterData {
+  /* 
+  Currently supports either all Llama or all gemini
+  TODO: refactor this as a list of types and values for each mediator(see design document)
+  */
   id: string;
-  apiKeys: APIKeyConfig;
```

Comment:
@dimits-ts @mkbehr @cjqian The original idea here was to have APIKeyConfig contain all API key info (as ExperimenterData may eventually contain non-API key settings). What do you think about keeping APIKeyConfig and nesting all api settings under there? E.g.,

```
APIKeyConfig {
  geminiApiKey: string;
  llamaApiKey: LlamaServerConfig;
  activeApiKeyType: ApiKeyType;
}
```

(we also may want `geminiApiKey` to point to a config, but I'm fine with waiting until we have something else to store there)